statistics/column.go

```diff
@@ -45,6 +45,9 @@ type Column struct {
 
 	// StatsLoadedStatus indicates the status of column statistics
 	StatsLoadedStatus
+	// PhysicalID is the physical table id,
+	// or it could possibly be -1, which means "stats not available".
+	// The -1 case could happen in a pseudo stats table, and in this case, this stats should not trigger stats loading.
 	PhysicalID int64
```

Comment:
will you use it heavily? i want to remove this field from the column/index struct to save memory.

Reply:
No.
I modified this just because this field has already been used for stats loading, and I'm changing the behavior of stats loading for the pseudo table.